server-base/src/main/java/org/apache/kylin/rest/controller/AdminController.java

```diff
@@ -118,11 +116,6 @@ public void cleanupStorage() {
         adminService.cleanupStorage();
     }
 
-    @RequestMapping(value = "/config", method = { RequestMethod.PUT }, produces = { "application/json" })
```

Comment:
I'm not quite sure how many users are using this API. If we can provide another way to enable it (on their own risk), that can be more flexible.

Reply:
Yes I aggree, it's better to enable or disable this API with kylin.properties configuration. I'm working on it.

Reply:
Currently the function that setting config from the front end only can be enable or disable at the backend by modify the property  "kylin.web.set-config-enable" at kylin.properties and it's disable by default. 
If "kylin.web.set-config-enable" is set to false, kylin will throw an exception when user try to  set config from website or restful API.
![image](https://user-images.githubusercontent.com/9884693/84350979-fd635b00-abec-11ea-9dfc-3eb884a64f7d.png)